jobserver/templates/project_release_list.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base.html" %}
+
+{% load humanize %}
+
+{% block content %}
+
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+
+    <li class="breadcrumb-item"><a href="/">Home</a></li>
+
+    <li class="breadcrumb-item">
+      <a href="{{ project.org.get_absolute_url }}">
+        {{ project.org.name }}
+      </a>
+    </li>
+
+    <li class="breadcrumb-item">
+      <a href="{{ project.get_absolute_url }}">
+        {{ project.name }}
+      </a>
+    </li>
+
+    <li class="breadcrumb-item active" aria-current="page">Releases</li>
+
+  </ol>
+</nav>
+
+
+<div class="row">
+  <div class="col-lg-8 offset-lg-2">
+
+    <p>
+      Below are all releases for the {{ project.name }} project across all of
+      its workspaces.
+    </p>
+    {% for release in object_list %}
+    <div class="border rounded p-2 pl-3 mb-3">
+
+      <div class="d-flex justify-content-between">
+        <span>
+          Release by {{ release.created_by.name }} in the
```

Comment:
s/Release by/File released by/?

I think avoiding noun-ing "Release", and speaking instead of "releasing files" and "released files", is better for user facing text, as users have no real context of A Release as a Noun.